src/redux/sagas.ts

```diff
@@ -1,38 +1,35 @@
-/* @flow */
-/* eslint flowtype/require-parameter-type: 0 */
 import { put, select, call, take, cancelled, fork } from 'redux-saga/effects';
-import { eventChannel } from 'redux-saga';
+import { eventChannel, Subscribe } from 'redux-saga';
 import { AppState, Platform } from 'react-native';
 import NetInfo from '@react-native-community/netinfo';
 import { networkSelector } from './createReducer';
 import checkInternetAccess from '../utils/checkInternetAccess';
 import { connectionChange } from './actionCreators';
-import type { HTTPMethod } from '../types';
+import DEFAULT_ARGS from '../utils/defaultConnectivityArgs';
```

Comment:
I just realised this could be added to the `constants.ts` file, so that all the defaults are compiled into one single place.